### ISSUE TITLE
Build ARM-compatible version of this Image

### ISSUE THREAD

**Issue by jcstein** [celestiaorg/local-celestia-devnet#11]
# On Mac (ARM)
```bash
docker run -p 26657:26657 -p 26659:26659 ghcr.io/celestiaorg/local-celestia-devnet:main
```

## Result
```bash
Unable to find image 'ghcr.io/celestiaorg/local-celestia-devnet:main' locally
2main: Pulling from celestiaorg/local-celestia-devnet
3docker: no matching manifest for linux/arm64/v8 in the manifest list entries.
4See 'docker run --help'.
```

[Link to snippet](https://app.warp.dev/block/Q3cMRBNCNKRJO8QLUyaoE2)

**Comment by jcstein** [comment]
The third line of the error message, docker: no matching manifest for linux/arm64/v8 in the manifest list entries., indicates that Docker is unable to find a matching manifest for the specified image and platform (Linux, architecture: arm64, variant: v8).

This error could occur because the image doesn't exist on the specified repository or the specified platform is not supported by the image. You can try pulling the image from a different repository, or you can specify a different platform that is supported by the image.

You can use the docker run command with the --platform option to specify the desired platform. For example:

```bash
docker run --platform linux/amd64 ghcr.io/celestiaorg/local-celestia-devnet:main
```

**Comment by smuu** [comment]
When specifying the amd64 platform on a mac, it does not run natively on Mac. Also, this approach is not possible for RPIs.

I propose reopening this issue and changing the `local-celestia-devnet` build to a multi-arch one.

See here: https://github.com/celestiaorg/.github/blob/main/.github/workflows/reusable_dockerfile_pipeline.yml#L97

**Comment by jcstein** [comment]
hmm, it worked for me on my Mac 🤔

**Comment by smuu** [comment]
Yes, modern Macs have an arm processor but can emulate amd processes.

**Comment by jcstein** [comment]
ah, cool. what are you on? I'd imagine intel macs can not use this `docker run --platform linux/amd64 ghcr.io/celestiaorg/local-celestia-devnet:main`

**Comment by jcstein** [comment]
also, I looked into the Dockerfile. I think it may be celestia-app that is not built correctly? do you know what the fix is for this?

**Comment by smuu** [comment]
Soon there will be a new common workflow in place, that will be used by celestia-app. Then this will also be built on arm.
Is this time sensitive?

**Comment by jcstein** [comment]
Nope, I have a note on the tutorials that says it's only been tested on AMD
